--- a/src/engine.c
+++ b/src/engine.c
@@ -1851,9 +1851,9 @@ ibus_hangul_engine_get_input_mode_symbol (IBusHangulEngine *hangul,
     IBusText **symbols = hangul->input_mode_symbols;

     if (symbols[0] == NULL) {
-        symbols[INPUT_MODE_HANGUL] = ibus_text_new_from_string ("한글");
+        symbols[INPUT_MODE_HANGUL] = ibus_text_new_from_string ("가");
         g_object_ref_sink(symbols[INPUT_MODE_HANGUL]);
-        symbols[INPUT_MODE_LATIN] = ibus_text_new_from_string ("영문");
+        symbols[INPUT_MODE_LATIN] = ibus_text_new_from_string ("A");
         g_object_ref_sink(symbols[INPUT_MODE_LATIN]);
     }
